dhis-2/dhis-support/dhis-support-hibernate/src/main/resources/org/hisp/dhis/usertype/UserTypes.hbm.xml

```diff
@@ -99,6 +99,10 @@
     <param name="clazz">java.util.ArrayList</param>
   </typedef>
 
+  <typedef class="org.hisp.dhis.hibernate.jsonb.type.JsonBinaryType" name="jbMap">
```

Comment:
Is Map a good candidate as JSONB entity? Should we have some wrapper around it?

Reply:
It's a very generic entity... I'm not sure, in the lower layers we are using `HttpHeaders` but don't want to use the web layer for this.